Dockerfile.awslambda

```diff
@@ -2,6 +2,7 @@ FROM public.ecr.aws/lambda/python:3.8
 
 COPY --from=frontend-awslambda /app/build ${LAMBDA_TASK_ROOT}/frontend/public
 
+COPY resources/attributions/docker-attributions.txt license.txt
```

Comment:
suggestion(non-blocking): why don't you do the same for the `license.txt` of `npm/python` dependencies?

It's a file used only for releases, so I think it's better to keep the `package.json` free from rarely used scripts (when developing).
To test the procedure locally you can then either run a `docker build` or copy the attributions file manually to the specific location.